README.md

```diff
@@ -59,7 +59,18 @@ Then fork and clone the repo, source setup, make the tests and run regression
 	fi
 	```
 
-9. Build the tests and run a regression simulation to prove everything is installed.  Building tests may take a while.
+9. Try compiling the HelloWally program and simulating it on the SystemVerilog with Verilator and on the Spike simulator.
+	``` cd examples/C/hello
```

Comment:
This line doesn't render in the Markdown without the following change:
```suggestion
	``` 
	cd examples/C/hello
```